tests/examplefiles/blazor.razor

```diff
@@ -0,0 +1,20 @@
+<div class="card" style="width:22rem">
+    <div class="card-body">
+        <h3 class="card-title">@Title</h3>
+        <p class="card-text">@ChildContent</p>
```

Comment:
The file has constructs that the lexer does not parse. It's fine if the lexer isn't exhaustive (this one appears quite minimal though), but can you at least make it so that everything in the test file is parsed correctly?